include/natalie/gc/allocator.hpp

```diff
@@ -8,6 +8,7 @@
 
 #include "natalie/gc/heap_block.hpp"
 #include "natalie/macros.hpp"
+#include "tm/hashmap.hpp"
 #include "tm/vector.hpp"
```

Comment:
is the vector include still needed then?

Reply:
Yep, this is still used for `m_free_blocks`. That might also make sense as a different data structure, but I haven't looked into it yet